docs/ui/insight/get-started.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Get started with RedisInsight
+linkTitle: Get started
+description: Get started with RedisInsight
+weight: 20
+
+---
+
+RedisInsight is available as a standalone download or [bundled with Redis Stack](/docs/stack/get-started/install). 
+Standalone downloads are available for Windows, MacOS, and Linux from the Redis Enterprise website and stores.
+
+## Install on Windows
+
+**From website**
+
+1. Download [RedisInsight](https://redis.com/redis-enterprise/redis-insight). 
+
+1. Open the `RedisInsight-v2-win-installer.exe` file.
+
+1. Open the RedisInsight icon.
+
+**From stores**
+
+* Go to [Microsoft Windows Apps](https://apps.microsoft.com/store/detail/redisinsight/XP8K1GHCB0F1R2) and download the RedisInsight app.
+
+## Install on MacOS
+
+**From website**
+
+1. Download [RedisInsight](https://redis.com/redis-enterprise/redis-insight). 
+
+1. Add RedisInsight to **Applications**. 
+
+**Using Homebrew**
+
+```
+brew install --cask redisinsight
+```
+
+To start RedisInsight, from your Terminal, run this command.
+
+```
+redisinsight
+``` 
+
+**From stores**
+
+* Go to [Apple Store](https://apps.apple.com/us/app/redisinsight/id6446987963) and download the RedisInsight app.
+
+
+## Install on Linux
+
+**From website**
+
+1. Download [RedisInsight](https://redis.com/redis-enterprise/redis-insight). 
+
+1. Open the Linux command line and install the package. 
+
+   * For Debian, run:
+
+     ```
+     sudo apt install ./RedisInsight-v2-linux-amd64.deb
+     ```
+
+   * For any common Linux distribution, use AppImage. 
+     - To make your downloaded file into an executable, run:
+
+        ```
+        chmod a+x RedisInsight-v2-linux-x86_64.AppImage
+        ```
+
+     - To start RedisInsight, open the Linux command line and navigate to the folder containing the downloaded file. Then, run:
+
+       ```
+       ./RedisInsight-v2-linux-x86_64.AppImage
+       ```
+
+**From stores**
+
+* You can install RedisInsight from [Snapcraft](https://snapcraft.io/redisinsight) or [FlatHub](https://flathub.org/apps/details/com.redis.RedisInsight) using instructions available at those sites.
+
+## Add a Redis database
+
+You can add a Redis database manually, create a new Redis Enterprise Cloud database, or connect to an existing database. To try Redis Enterprise Cloud, select **Try Redis Cloud with enhanced database capabilities**.
+
+If you already have Redis database up and running locally, select **Autodiscover Databases**.
```

Comment:
If you already have Redis databases up and running locally, RedisInsight will automatically search for it and add it if found.

Reply:
users can't manually trigger the auto-discovery for local databases